base/pop/com/popenv.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
No. popenv.sh is part of the runtime. Even worse, it is always sourced and never run (there would be no point in running it).

Reply:
I've removed them, and adapted the test script to only check for scripts with an exec bit set.